php/src/Types.cpp

```diff
@@ -1218,13 +1218,11 @@ IcePHP::StructInfo::optionalFormat() const
 bool
 IcePHP::StructInfo::usesClasses() const
 {
-    for (DataMemberList::const_iterator p = members.begin(); p != members.end(); ++p)
-    {
-        if ((*p)->type->usesClasses())
+    for (const auto& dm : members)
```

Comment:
we always write for loops with curly braces in C++

Reply:
Yes, a typo on my part.